jac-uk/admin
# #996 Candidate search

NOTE: once this has been tested and approved before it is merged the following tasks should be addressed:
- [x] Update Table references to use JAC Kit
- [x] Update all references to JAC Kit tableQuery. It has now moved from helpers to Table component

---
## What's included?
Updates tables to use latest version of candidate search, which enables matches on:
 - email
 - application reference number (NOTE: we concentrate on the part after -)
 - candidate name parts (e.g. for "joh" and "sm" will match candidate "John Smith")
 
## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
The following views have been updated. All should now enable improved candidate search:
 - Candidates
 - Applications
 - Review
 - Independent assessments
 - Qualifying test reponses

## Risk - how likely is this to impact other areas?
🟠 Medium risk - this does change code that is shared with other areas

## Additional context
Candidate search by part of name:
<img width="1031" alt="image" src="https://user-images.githubusercontent.com/8524401/116068400-20babb80-a682-11eb-888c-a174e7c12cb2.png">

Application search by reference number:
<img width="623" alt="image" src="https://user-images.githubusercontent.com/8524401/116068511-392ad600-a682-11eb-8655-c465ad1cf20b.png">

Application search by partial reference number:
<img width="684" alt="image" src="https://user-images.githubusercontent.com/8524401/116068733-7727fa00-a682-11eb-9463-0b9865dde195.png">

---
PREVIEW:DEVELOP  _can be OFF, DEVELOP or STAGING_
